### PR TITLE
Hope/help command

### LINKED ISSUE #66: Add a "help" command to the bot
See title. It should list all the commands, each with a description and usage

### LINKED ISSUE #71: Add helpful text to bot's `sendMoney` response
How to use `sendMoney` isn't really all that clear. At minimum, should probably:

- [ ] Have the bot say "check ur dm" or something like that so the user knows to check their DM for the link
- [ ] Add some instructional text to the user's DM that makes it clear the user needs to click the link to send the tx

However, we might also want to somehow have the bot react to the message containing the link or delete it or send another message or some other way of letting the user know not to click old links in the user's DM with the bot and replay attack themselves. This may very well belong in a different issue, as a generic replay-attack defense mechanism for applications like the discord bot that don't have access to the user's keys would be really nice.

### PR DESCRIPTION
**Pull request by hopeyen**
Added the help command as well as instructional text for sendMoney

### CODE REVIEW

File: apps/bot/src/commands/Help.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      name: 'help',
+      description: 'Help commands bot.',
+      category: 'Information',
+      usage: client.settings.prefix.concat('help'),
+      cooldown: 1000,
+      requiredPermissions: ['SEND_MESSAGES'],
+    });
+  }
+
+  public async run(message: Message): Promise<void> {
```
Comment (Sladuca) [inline apps/bot/src/commands/Help.ts:17]: Please fmt with `nx format`. This reminds me I need to add it as a pre-commit hook

File: apps/bot/src/config/config.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ import { BotSettings } from '../types';
 export const settings: BotSettings = {
   presence: {
     activity: {
-      name: 'Just HODL',
-      type: 'PLAYING',
+      name: 'AoT',
```
Comment (Sladuca) [inline apps/bot/src/config/config.ts:6]: *chef's kiss*

File: apps/frontend/src/components/ApproveRedirect.svelte [hunk 3]
```diff
@@ -5,14 +5,18 @@
   import AmountFormatter from '@baf-wallet/base-components/AmountFormatter.svelte';
   import { BafError } from '@baf-wallet/errors';
   import Spinner from 'svelte-spinner';
-
+
```
Comment (Sladuca) [inline apps/frontend/src/components/ApproveRedirect.svelte:8]: why is the frontend code being modified here? Did you make this branch off of `main` or `hope/formValidation`?

**Review by Sladuca — CHANGES_REQUESTED** [review]
Nice job! A bit confused why frontend code is changing for this. Perhaps this branch is based off of `hope/formValidation` instead of `main`?

File: apps/frontend/src/components/ApproveRedirect.svelte (hunk 3 above)
Comment (hopeyen) [reply apps/frontend/src/components/ApproveRedirect.svelte:8]: hope/formValidation. Now I see that I should've used `main`

**Comment by Sladuca** [comment]
Oh I forgot to mention this got implemented elsewhere while you were on vacation - seriously no hard feelings :).

**Pull request closed without merging**